pkg/dashboard/assets/js/namespace_list.js

```diff
@@ -0,0 +1,37 @@
+$(function () {
+    var $namespaceCards = $("#namespaceCards .card")
+    var $searchBox = $("#search")
+
+    // filters namespace cards containing the search text
+    function filterNamespaceCards() {
+        var searchText = $searchBox.val()
+        $namespaceCards.filter(function() {
+            var cardNamespace = $("strong", $(this)).text()
+            var cardContainsSearchText = cardNamespace.toLowerCase().indexOf(searchText.toLowerCase()) > -1
+            if(cardContainsSearchText) {
+                $(this).show()
+            } else {
+                $(this).hide()
+            }
+        })
+    }
+
+    // clicks on the "view" button for the first visible namespace card
+    function viewDashboardForFirstNamespace() {
+        $namespaceCards.filter("div:not([style='display: none;'])").first().find("a[data-name='view'")[0].click()
+    }
+
+    // filter namespace cards on every key typed
+    $searchBox.on("keyup", filterNamespaceCards)
+
+    // visit the first filtered namespace card's view/dashboard page by pressing enter
+    $searchBox.on("keypress", function(e) {
+        var key = e.which;
+        if(key == 13) { // enter
+            viewDashboardForFirstNamespace()
+        }
+    })
```

Comment:
This behaviour is still up for discussion. See the original discussion here: https://github.com/FairwindsOps/goldilocks/pull/195#discussion_r468875157

@sudermanjr To quote the last part of the discussion and bring it over to this PR:

> We can drop the "visit first namespace on enter" behaviour entirely if you'd like. Then `<enter>` would effectively do nothing and would instead just rely on the filtering. It would be like an illusion of enter.
--Harrison

Thoughts?

Reply:
I think I would rather that than the way it is currently implemented. Other options would be to add a message that tells you what's happening, or maybe highlight the top result to indicate "selection".

To be 100% honest and up-front, this section is most likely going to get re-written in the near future anyway, so dropping it might be best.

Reply:
Sure no worries! I'll remove this feature entirely as it's really not needed :)

Reply:
thanks!